website/templates/util/render_node.mako

```diff
@@ -51,16 +51,26 @@
             % endif
             </span>
 
-            <!-- Show/Hide recent activity log -->
-            % if not summary['archiving']:
             <div class="pull-right">
                 % if not summary['primary'] and 'write' in user['permissions'] and not node['is_registration']:
                     <i class="fa fa-times remove-pointer" data-id="${summary['id']}" data-toggle="tooltip" title="Remove link"></i>
                     <i class="fa fa-code-fork" onclick="NodeActions.forkPointer('${summary['id']}', '${summary['primary_id']}');" data-toggle="tooltip" title="Fork this ${summary['node_type']} into ${node['node_type']} ${node['title']}"></i>
                 % endif
-                <i id="icon-${summary['id']}" class="pointer fa fa-angle-down" onclick="NodeActions.openCloseNode('${summary['id']}');" style="font-weight:bold;"></i>
```

Comment:
I just noticed this old line has an onclick function. Can you look into doing something similar to replace your class level onclick binding?